test/linear_samplers/proc_block_row_rand_replace.jl

```diff
@@ -0,0 +1,34 @@
+# This file is part of RLinearAlgebra.jl
+# This file was written by Nathaniel Pritchard  
+
+module ProceduralTestLSBRReplace
+
+using Test, RLinearAlgebra, Random
+
+import LinearAlgebra: norm
+
+Random.seed!(1010)
+
+@testset "LSBR Random Sampling with Replacement -- Procedural" begin
+
+    # Verify appropriate super type
+    @test supertype(LinSysBlkRowReplace) == LinSysBlkRowSampler
+
+    # Test whether row ordering remains fixed
+    A = rand(10,6)
+    b = rand(10)
+    x = rand(6)
+
+    cyc = LinSysBlkRowReplace()
+
+    v, M, res = RLinearAlgebra.sample(cyc, A, b, x, 1)
+
+    for j = 2:5
+        v, M, res = RLinearAlgebra.sample(cyc, A, b, x, j)
+        @test norm(res - (v * A * x - v * b)) < eps() * 1e2
+    end
+
```

Comment:
Need to test assertions are correctly triggered.

Reply:
I have added an assertion test for the two assertions